src/scripts/process_Institution.py

```diff
@@ -257,8 +262,8 @@ def __compare_place_and_start_for_Inst(index, row, inst_wiki, l, today, wikidata
         if wikidata_id != row["wikidata_id"]:
             row["wikidata_id"] = wikidata_id
             message = (
-                "For row %s, the wikidata_id can be queried. You should look institution up in Wikidata and write the wikidata_id in your table in the future. "
-                % index
+                "For row %s, the wikidata_id can be queried. You should look institution up in Wikidata and write "
+                "the wikidata_id in your table in the future. " % index
```

Comment:
I don’t getting it what does it mean that a qid is in the future. Might want to rephrase.

Reply:
It means for this Institution named entity, if the user query it, actually there is a qid. So that it is a suggestion to query first and add qid into table then. Rephrased.